showcase/BUILD.bazel

```diff
@@ -19,14 +19,16 @@ proto_library_with_info(
         "@com_google_gapic_showcase//schema/google/showcase/v1beta1:showcase_proto",
 #        "//showcase/gapic-showcase-extended/proto:showcase_proto_extended",
         "@com_google_googleapis//google/cloud:common_resources_proto",
+        "@com_google_googleapis//google/cloud/location:location_proto"
     ],
 )
 
 java_proto_library(
     name = "showcase_java_proto",
     deps = [
         "@com_google_gapic_showcase//schema/google/showcase/v1beta1:showcase_proto",
-#        "//showcase/gapic-showcase-extended/proto:showcase_proto_extended",
+#       "//showcase/gapic-showcase-extended/proto:showcase_proto_extended",
+        "@com_google_googleapis//google/cloud/location:location_proto"
```

Comment:
Wondering if this is needed in the bazel target. Looking at an example bazel rule in googleapis with location mixin doesn't seem to include it in `java_proto_library`: https://github.com/googleapis/googleapis/blob/a32790cd647c34be01845e0edd580557bced3d42/google/cloud/tpu/v2/BUILD.bazel#L59-L62

Reply:
Thanks for pointing this out! You're right, looks like it is not needed. I will remove it.

Reply:
Hm surprisingly, also removing these two deps (https://github.com/googleapis/googleapis/blob/a32790cd647c34be01845e0edd580557bced3d42/google/cloud/tpu/v2/BUILD.bazel#L79-L85) didn't seem to make a difference to the client re-generation.

Reply:
Ah interesting! Let me try locally